docs/framework-jrebel_agent.md

```diff
@@ -1,31 +1,32 @@
 # JRebel Agent Framework
-The JRebel Agent Framework causes an application to be automatically configured to work with an IDE using [JRebel][].
+
+This framework enables the use of [JRebel][jrebel] with deployed applications. Pushing any [JRebel Cloud/Remote][remoting] enabled application (containing `rebel-remote.xml`) will automatically download the latest version of [JRebel][jrebel] and set it up for use.
 
 <table>
   <tr>
     <td><strong>Detection Criterion</strong></td>
-    <td>Existence of the `rebel.xml` and `rebel-remote.xml` files in either the root or `WEB-INF/classes` directory or the application.</td>
+    <td>Presence of a <tt>rebel-remote.xml</tt> file inside the application archive. This file is present in every application that is configured to use <a href="http://manuals.zeroturnaround.com/jrebel/remoting/index.html" target="_blank">JRebel Cloud/Remote</a>.</td>
```

Comment:
I realize this is a minor quibble, but I'd like this [match the other documentation](https://github.com/cloudfoundry/java-buildpack/blob/master/docs/framework-app_dynamics_agent.md) a bit more.  Please start it with "Existence of a...".